sys/kern/subr_intr.c

```diff
@@ -901,19 +934,29 @@ intr_pic_claim_root(device_t dev, intptr_t xref, intr_irq_filter_t *filter,
 	 * Note that we further suppose that there is not threaded interrupt
 	 * routine (handler) on the root. See intr_irq_handler().
 	 */
-	if (intr_irq_root_dev != NULL) {
+	KASSERT(rootnum < INTR_ROOT_NUM,
+		("%s: invalid interrupt root %d", __func__, rootnum));
```

Comment:
I believe style is this should be tab, then 4 spaces.

Reply:
done